### PR TITLE
Docs for 0.29 (WIP)

### LINKED ISSUE #6940: Documentation of Row Charts is missing in the Users Guide
The super useful *Row Chart* (aka 📊 +90° for added category label reading comfort 😉) implemented in #4317 apparently never got any honorable mention in [the users guide](https://metabase.com/docs/latest/users-guide/05-visualizing-results.html)

Apart from the obvious advantage that you don't have to turn your head and halfway break your neck to read the labels as you do on a bar chart (if it has vertical labels) I think it also deserves pointing out that by default it also adds an `Others` bin. This works really well if you apply "Sort by count, descending" in the `...` menu as it, like the pie chart highlights the "long tail" 🦕 of your data. It includes auto-magic responsive adjustment of the number of categories to fit the available space.

 rowyourboat

### PR DESCRIPTION
**Pull request by mazameli**
Also added documentation for Row charts, so this fixes #6940

### CODE REVIEW

File: docs/users-guide/08-dashboard-filters.md [hunk 1]
```diff
@@ -58,6 +57,24 @@ Once you’ve added a filter to your dashboard, just click on it to select a val
 
 ![Using a filter](images/dashboard-filters/08-use-filter.png)
 
+### Choosing between a dropdown or autocomplete for your filter
+
+As of Metabase 0.29, we've greatly improved how filtering works with fields that have lots of values. If the field you're using for a filter has more than 100 unique values, you'll now automatically see a search box with autocomplete suggestions.
```
Comment (salsakran) [inline docs/users-guide/08-dashboard-filters.md]: Not sure we should call out a specific version in the docs? I.e. I should be able to read a given versions docs and get info on that version.

File: docs/users-guide/10-pulses.md [hunk 2]
```diff
@@ -1,25 +1,29 @@
 
-## Sharing Updates with Pulses
-The Pulses feature in Metabase gives you the ability to automatically send regular updates to your teammates to help everyone keep track of changes to the metrics that matter to you most. You can deliver a pulse via email or [Slack](https://slack.com/), on the schedule of your choice.
+## Sharing updates with pulses
+The Pulses feature in Metabase gives you the ability to automatically send regular updates to your teammates to help everyone keep track of changes to the metrics that matter to you most. You can deliver a ulse via email or [Slack](https://slack.com/), on the schedule of your choice.
```
Comment (salsakran) [inline docs/users-guide/10-pulses.md]: `ulse` got a `p` dropped somewhere

File: docs/users-guide/10-pulses.md [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
+### Pick your data
+Before you can create a pulse, you’ll need to have some [saved questions](06-sharing-answers.md). Click the dropdown to see a list of all your saved questions. You can type in the dropdown to help filter and find the question you’re looking for.
 
 ![Pick your data](images/pulses/03-pick-your-data.png)
 
 When you select a saved question, Metabase will show you a preview of how it’ll look in the pulse. Because of the space constraints of email and Slack, Metabase will automatically make some adjustments to the appearance of your saved question so that it looks great in the pulse. For example, in order to save space, pie charts will automatically be transformed into bar charts.
 
 ![Behold! The metamorphosis.](images/pulses/04-transformation.png)
 
+As of Metabase 0.29, you can now include tables in your pulses. They'll be capped to 10 columns and 20 rows, and for emailed pulses the rest of the results will be included automatically as a file attachment, with a limit of 2,000 rows.
```
Comment (salsakran) [inline docs/users-guide/10-pulses.md]: I don't think calling out a specific version is a good idea.

**Review by salsakran — CHANGES_REQUESTED** [review]
Minor changes, see comments.

**Comment by mazameli** [comment]
Thanks Sameer. Fixed those comments. I need to add a couple of x-ray images and then this should be good to go.

**Comment by mazameli** [comment]
Any objections to me merging this?

**Review by salsakran — APPROVED** [review]

**Comment by salsakran** [comment]
go fer it.

**Pull request merged**